README.md

```diff
@@ -119,3 +112,34 @@ To only run integration tests:
 ```bash
 npm run test:integration
 ```
+
+## Configurations
+
+ENVS:
+
+SERVER_PORT set the server port number - deafult to 8080
+
+LOG_LEVEL set the log level *based on 'winston' logger, available values as declared in winston logger docs, default to 'info'
```

Comment:
@map-colonies/js-logger is not 'winston'

Reply:
fixed it aa489ae